packages/outline-core-link/src/outline-core-link.ts

```diff
@@ -24,9 +28,15 @@ import componentStyles from './outline-core-link.css.lit';
  * @cssprop --outline-core-link-color-focus: The link color when focusing on the link.
  * @todo - Add support for outline/ring on the focus state of the link.
  */
-@customElement('outline-core-link')
+@customElement(componentName)
 export class OutlineCoreLink extends OutlineElement {
   static styles: CSSResultGroup = [componentStyles];
+
+  /**
+   * If the link is slotted, we'll test, modify & render the slotted link instead of the linkHref and linkText properties.
+   */
+  @state() isSlottedLink = false;
+
```

Comment:
This seems to be an unused code.
The only other code that has `isSlottedLink`, is setting it to `true`.